config.go

```diff
@@ -29,10 +30,18 @@ type Config struct {
 	onChangeCallbacks   []func(cfg *Config)
 }
 
-type ConfigMap map[string]map[string]string
+type ConfigMap struct {
+	mu sync.RWMutex
+	m  map[string]map[string]string
+}
 
-func (a *App) getConfigFilename(forceCurrentWorkingDir bool) string {
+func NewConfigMap() ConfigMap {
```

Comment:
I guess it can, I didn't think it would make any functional difference with the methods being pointer receivers (only bug I could foresee was inadvertently taking a copy of the mutex), and the values are small. Wasn't sure! I'll will switch to a pointer since it made you ask.